monkey/tests/unit_tests/monkey_island/cc/services/authentication_service/test_authentication_service.py

```diff
@@ -30,13 +31,20 @@ def mock_island_event_queue(autouse=True) -> IIslandEventQueue:
     return MagicMock(spec=IIslandEventQueue)
 
 
+@pytest.fixture
+def mock_user_datastore(autouse=True) -> MongoEngineUserDatastore:
```

Comment:
We can use `UserDatastore` here, since that's what `AuthenticationFacade` expects